src/pybind/mgr/cephadm/module.py

```diff
@@ -1253,12 +1252,8 @@ def _create_osd(self, all_hosts_, drive_group):
                     continue
 
                 # create
-                ret, keyring, err = self.mon_command({
```

Comment:
Just to clarify. Moving this doesn't fix any bugs, right?

Reply:
right, just refactoring a bit